libs/io_repl.c

```diff
@@ -95,8 +97,13 @@ void b_putchar(int idx, int *cells) {
 }
 
 void b_float_print(int idx, int *cells) {
-  float value = cells[idx] / 100.0;
-  printf("%.2f", value);
+//  float value = cells[idx] / 100.0;
+//  printf("%.2f", value);
+printf(
```

Comment:
hahaha! You're right!
I'm actually overriding this command inside the `io_repl.c`, if you check it [here](https://github.com/brain-labs/brain/pull/38/files/ef41300acca49574279fea8e82f7fdf1399e425b#diff-f43bc14b850b7f7f2e203ea08b753e7dR115) you will see that the __REPL__ overrides the `b_debug` method.. I'm using this other method to print the real debug information of the Brain tape rather than the future `$` inside the Brain REPL